facebookresearch/ssvp_slt
# Adding documentation for installing dlib on Windows

### Summary:
This PR adds documentation detailing the errors encountered when running the project on a Windows 11 environment. It includes steps to reproduce the issues, full error logs, and suggested solutions.

### Changes Included:
- Added a new documentation file: `docs/windows_testing_errors.md`.

### How to Test:
1. Follow the steps documented to reproduce the error on a Windows machine.
2. Apply the suggested fixes and verify that the script runs without errors.

### Additional Notes:
- This is part of an ongoing effort to ensure the project runs smoothly across different operating systems.
- Further testing on various Windows versions may be necessary.
